cli/include/CMakeLists.txt

```diff
@@ -16,4 +16,6 @@ else()
 endif()
 
 add_subdirectory(gz/utils)
-install(DIRECTORY ignition DESTINATION ${IGN_INCLUDE_INSTALL_DIR_FULL})
\ No newline at end of file
+install(DIRECTORY ignition DESTINATION ${IGN_INCLUDE_INSTALL_DIR_FULL})
+install(DIRECTORY external-cli/ignition DESTINATION ${IGN_INCLUDE_INSTALL_DIR_FULL})
+install(DIRECTORY vendored-cli/ignition DESTINATION ${IGN_INCLUDE_INSTALL_DIR_FULL})
```

Comment:
two problems:

* only one of these should be installed, never both, so this `install` logic needs to be inside the if/else block
* the `external-cli` and `vendored-cli` folders should never be part of the install path. I'm guessing we should add a `CMakeLists.txt` inside each subfolder that contains `install(DIRECTORY ignition ...` and `add_subdirectory(gz/utils)`

Reply:
Branch rebased, install logic fixed

https://github.com/ignitionrobotics/ign-utils/pull/55/commits/50f8dbde9bdf3fe5237a5845d6127e48c41a42e3

CI is green!